x-pack/platform/plugins/shared/lens/public/app_plugin/shared/edit_on_the_fly/helpers.ts

```diff
@@ -70,7 +70,11 @@ export const getGridAttrs = async (
 
   const dataView = dataViewSpec
     ? await data.dataViews.create(dataViewSpec)
-    : await getESQLAdHocDataview(query.esql, data.dataViews, { skipFetchFields: true });
+    : await getESQLAdHocDataview({
```

Comment:
The suggestions api only uses the indexPatterns from the dataViews and not the fields or the timeField